src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.zh-Hans.xlf

```diff
@@ -69,17 +69,17 @@
       </trans-unit>
       <trans-unit id="Assign_out_parameters">
         <source>Assign 'out' parameters</source>
-        <target state="translated">分配 "out" 参数</target>
+        <target state="translated">赋值 "out" 参数</target>
```

Comment:
I've changed several translations of "assign". This should be a terminology in computer science that has standard translation.